datasets/panstarrs.yaml

```diff
@@ -0,0 +1,24 @@
+Name: Pan-STARRS PS1 Survey
+Description: |
+  The PS1 surveys used a 1.8 meter telescope and its 1.4 Gigapixel camera to image the sky in five broadband filters. The largest of these surveys provides coverage of the entire sky north of -30 degrees declination.
+Documentation: https://outerspace.stsci.edu/display/PANSTARRS/
+Contact: archive@stsci.edu
+ManagedBy: "[Space Telescope Science Institute](http://www.stsci.edu/)"
+Citation: Please see the documentation for full citation instructions.
+UpdateFrequency: Never
+Tags:
+  - astronomy
+License: STScI hereby grants the non-exclusive, royalty-free, non-transferable, worldwide right and license to use, reproduce, and publicly display in all media data from the PS1 surveys.
+Resources:
+  - Description: PS1 DR1 and DR2 files
```

Comment:
They are actually image files (and specifically FITS files).  But in the future we could add some other PS1 data such as the bulk catalog csv files.  I'll leave it up to you whether we should be more specific about the data description.

Reply:
Oops, yes -- I will specify that we are including image files.